docs/.vitepress/theme/components/OldDocument.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="old-document">
+    <p>
+      This documentation describes <b>the old</b> Vite version. For the latest
+      version, see
+      <a href="https://vite.dev/" class="new-document-link"
+        >the latest document</a
+      >.
+    </p>
```

Comment:
Nitpicking, maybe the text can be something like:
```suggestion
    <p>
      This documentation covers Vite 5 <b>(old version)</b>. For the latest
      version, see
      <a href="https://vite.dev" class="new-document-link"
        >https://vite.dev</a
      >.
    </p>
```

The ideal text for me is something like "This documentation covers Vite 5. For the latest version (Vite 6), see https://vite.dev", but the Vite 6 part is a little hard to maintain and update.